test/integration/scheduler/preemption_test.go

```diff
@@ -1314,3 +1314,155 @@ func TestPDBInPreemption(t *testing.T) {
 		})
 	}
 }
+
+// wrapNext wraps the NextPod() method to make it appear the preemption has been done already and the nominated node has been set.
+func wrapNext(f func() *framework.QueuedPodInfo) func() *framework.QueuedPodInfo {
+	return func() (podInfo *framework.QueuedPodInfo) {
+		podInfo = f()
+		podInfo.Pod.Status.NominatedNodeName = "node-1"
+		return podInfo
+	}
+}
+
+func initPreemptorTest(t *testing.T, nsPrefix string, opts ...scheduler.Option) *testutils.TestContext {
```

Comment:
Choose a better name. This is not about preemption.

Maybe you can also do:

`test.Scheduler.NextPod = func() {`

So that we don't leave standalone functions that other tests don't need.

Reply:
Done